cmd/tether/attach.go

```diff
@@ -87,6 +92,21 @@ func (t *attachServerSSH) Reload(config *tether.ExecutorConfig) error {
 	return nil
 }
 
+// Enabled sets the enabled to true
+func (t *attachServerSSH) Enabled() {
+	atomic.StoreInt32(&t.enabled, 1)
```

Comment:
Do we need this given that you grab the attachServerSSH lock when you change enabled in start() and stop() below?